src/decoder_midigeneric.cpp

```diff
@@ -25,6 +25,10 @@
 
 constexpr int GenericMidiDecoder::midi_default_tempo;
 
+// 1 ms of MIDI message resolution for a 44100 Hz samplerate
+constexpr int samples_per_play = 512;
+constexpr int bytes_per_sample = sizeof(int_least16_t) * 2;
```

Comment:
Why `int_least16_t`? This could be larger than 16 bit.

Why not `int16_t`?

Reply:
Because fmmidi used it, was a copy-paste from there :sweat_smile: . But you are right, this makes no sense.

Reply:
fixed